.env

```diff
@@ -0,0 +1 @@
+STORYBOOK_MAPBOX_API_TOKEN=pk.eyJ1IjoibGF5ZXItbWFuYWdlciIsImEiOiJjazA2dTJnMG0zc3lwM251dGM0aXhiYTVxIn0.AaoErQTRezfQshqnEKIg8A
```

Comment:
please... let's turn our eyes and look somewhere else and did like nothing happened